File: library/src/main/java/com/getstream/sdk/chat/rest/response/TokenResponse.java
package com.getstream.sdk.chat.rest.response;

import com.getstream.sdk.chat.rest.User;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class TokenResponse {
    @SerializedName("user")
    @Expose
    private User user;

    @SerializedName("access_token")
    @Expose
    private String token;

    public User getUser() {
        return user;
    }

    public String getToken() { return token; }
}

File: client/src/main/java/io/getstream/chat/android/client/api/models/TokenResponse.kt
package io.getstream.chat.android.client.api.models

import com.google.gson.annotations.SerializedName
import io.getstream.chat.android.client.models.User

internal data class TokenResponse(
    val user: User,
    @SerializedName("access_token")
    val accessToken: String
)
